manifests/0000_50_olm_07-olm-operator.deployment.ibm-cloud-managed.yaml

```diff
@@ -24,6 +24,38 @@ spec:
       serviceAccountName: olm-operator-serviceaccount
       priorityClassName: "system-cluster-critical"
       containers:
+        - name: package-server-manager
+          command:
+          - /bin/psm
+          - start
+          args:
+          - --name
+          - $(PACKAGESERVER_NAME)
+          - --namespace
+          - $(PACKAGESERVER_NAMESPACE)
+          - --metrics
+          - :8083
+          image: quay.io/operator-framework/olm@sha256:de396b540b82219812061d0d753440d5655250c621c753ed1dc67d6154741607
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: PACKAGESERVER_NAME
+            value: packageserver
+          - name: PACKAGESERVER_IMAGE
+            value: quay.io/operator-framework/olm@sha256:de396b540b82219812061d0d753440d5655250c621c753ed1dc67d6154741607
```

Comment:
A quick note on this as I didn't want to add a comment to the YAML manifest - the CVO currently exposes a way to override default container images through an image-references file, which is mounted in a component's container image, merged at release payload build time, and resolved at startup. In order to reference the packageserver container image without having to query on-demand for that information or saving that state to a k8s resource, we can use an environment variable that the CVO will essentially sed if the image specified matches the image detailed in the image-references file.